script/BaseScript.s.sol

```diff
@@ -11,11 +11,20 @@ import {Script} from "lib/forge-std/src/Script.sol";
 import {Utils} from "script/Utils.sol";
 import {ActorAddresses} from "script/Actors.sol";
 import {ContractAddresses} from "script/ContractAddresses.sol";
+import {TransparentUpgradeableProxy} from "lib/openzeppelin-contracts/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import {ProxyAdmin} from "lib/openzeppelin-contracts/contracts/proxy/transparent/ProxyAdmin.sol";
 import {console} from "lib/forge-std/src/console.sol";
 
+
 abstract contract BaseScript is Script, Utils {
     using stdJson for string;
 
+    struct ProxyAddresses {
```

Comment:
load up all data from JSON file into this struct.

This way you can assert that the JSON matches the on-chain for the trinity:

Proxy - ProxyAdmin - Implementation